File: AccessoDatos/app/src/main/java/org/institutoserpis/juanminm/accessodatos/ClientesSQLiteHelper.java
package org.institutoserpis.juanminm.accessodatos;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

public class ClientesSQLiteHelper extends SQLiteOpenHelper {

    String cadSQL = "CREATE TABLE Clientes (codigo INTEGER, nombre TEXT, telefono TEXT)";

    public ClientesSQLiteHelper(Context context, String name, SQLiteDatabase.CursorFactory factory, int version) {
        super(context, name, factory, version);
    }

    @Override
    public void onCreate(SQLiteDatabase db) {
        db.execSQL(cadSQL);
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
        db.execSQL("DROP TABLE IF EXISTS Clientes");
        db.execSQL(cadSQL);
    }
}

File: KAccessoDatos/app/src/main/java/org/institutoserpis/juanminm/kaccessodatos/ClientesSQLiteHelper.kt
package org.institutoserpis.juanminm.kaccessodatos

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper

/**
 * Created by juamar on 16/01/18.
 */

class ClientesSQLiteHelper(
        context: Context?,
        name: String?,
        factory: SQLiteDatabase.CursorFactory?,
        version: Int
) : SQLiteOpenHelper(context, name, factory, version) {
    private val cadSQL: String = "CREATE TABLE Clientes (codigo INTEGER, nombre TEXt, telefono TEXT)"

    override fun onCreate(db: SQLiteDatabase) {
        db.execSQL(cadSQL);
    }

    override fun onUpgrade(db: SQLiteDatabase, oldVersion: Int, newVersion: Int) {
        db.execSQL("DROP TABLE IF EXISTS Clientes")
        db.execSQL(cadSQL);
    }

}